src/test/java/com/issoft/training/tests/AutomationPracticeShoppingTests.java

```diff
@@ -0,0 +1,188 @@
+        accountInformationPage.inputAddressState(ConfProperties.getProperty("state"));
+        accountInformationPage.inputAddressPostCode(ConfProperties.getProperty("postcode"));
+        accountInformationPage.inputCompanyCountry(ConfProperties.getProperty("country"));
+        accountInformationPage.inputPhoneMobile(ConfProperties.getProperty("phone"));
+        accountInformationPage.inputAddressAlias(ConfProperties.getProperty("alias"));
+        accountInformationPage.clickSubmitAccountBtn();
+
+        // Expected result: Account was created
+        Assert.assertTrue(driver.findElement(Constants.HEADER_MY_ACCOUNT_PAGE).isDisplayed());
+        ConfProperties.setProperty("new_email_registered", "yes");
+        // Sign out
+        driver.findElement(Constants.MY_ACCOUNT_SIGN_OUT).click();
+    }
+
+    @Test( priority = 1, groups = {"loginTests", "checkinTests"},
+            description = "Verify the ability to login an account.")
+    @Severity(SeverityLevel.CRITICAL)
+    @Description("Test Description: Verify the ability to login an account.")
+    @Story("Story name: Account login.")
+    public void loginMyAccountTest() {
+        // test AP-2
+        driver.get(getProperty("url_automationpractice_loginpage"));
+
+        loginAccountPage = new LoginAccountPage(driver);
+        Assert.assertEquals(driver.getTitle(), Constants.TITLE_OF_ACCOUNT_PAGE);
+
+        loginAccountPage.inputLoginEmail(ConfProperties.getProperty("email"));
+        loginAccountPage.inputLoginPasswd(ConfProperties.getProperty("passwd"));
+        getScreenShotFile(driver, Constants.LOGIN_EMAIL_SCREEN);
+        loginAccountPage.clickLoginAccountBtn();
+
+        // Expected result: You were able to login
+        Assert.assertTrue(driver.findElement(Constants.HEADER_MY_ACCOUNT_PAGE).isDisplayed());
+    }
+
+    @Test( priority = 2, groups = {"wishlistTests", "checkinTests"},
+            description = "Verify the ability to add a product to auto-created Wishlist.")
+    @Severity(SeverityLevel.NORMAL)
+    @Description("Test Description: Verify the ability to add a product to auto-created Wishlist.")
+    @Story("Story name: Add a product to Wishlist.")
+    public void AddProductToAutoWishlistTest() {
+        // test AP-3
+        myAccountPage = new MyAccountPage(driver);
+        myWishlistsPage = new MyWishlistsPage(driver);
+
+        driver.get(getProperty("url_my_account_page"));
+        Assert.assertTrue(driver.findElement(Constants.HEADER_MY_ACCOUNT_PAGE).isDisplayed());
+
+        myAccountPage.clickMyWishListBtn();
+        Assert.assertTrue(myWishlistsPage.isDisplayedMyWishListHeader());
+
+        myWishlistsPage.clickFirstProductDetailsLink();
+        Assert.assertTrue(myWishlistsPage.isDisplayedMyWishListBtn());
+
+        // Expected result: Wishlist was created automatically and your product is in the list
+        myWishlistsPage.clickMyWishListBtn();
+        Assert.assertTrue(myWishlistsPage.isDisplayedProductAddedAlert());
```

Comment:
No need to assert each step. The good rule for the test is that it has one assert. It possible to have more than one assert, but it should check the same state(page). 
In your case if you want to check different pages it sounds good to split the current test on a smaller parts.

According to the scenario in the task you should check the final state. For the current test it will be that product was added tot the wish list

Reply:
Checked the final state only.